en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -586,8 +585,8 @@ notation::
         }
     ]);
 
-In the above example, you'll still get authors even if they don't have
-a published profile. To only get authors with a published profile use
+In the above example, you'll still get articles even if the author doesn't have
```

Comment:
this should be `authors` IMO

Reply:
You want authors to be `authors`?

Reply:
Authors is the data not the table here so I think just 'authors' works.

Reply:
`you'll still get articles even....` => `you'll still get authors even....`

Reply:
Got it. Reverted typo.